web/src/components/button/CardButton/index.tsx

```diff
@@ -443,7 +444,8 @@ export function ButtonOwnerPhone({ user, button }) {
   }
   return (
     <>
-      {user?.publishPhone && (
+      {(user?.publishPhone || 
+        isAdmin(sessionUser)) && (
```

Comment:
@watchoutfreedom  only admins can see the published phones?

Reply:
If the user doesnt share it yes